packages/core/src/lib/template-engine/built-in-extensions/query-builder/reqTagRunner.ts

```diff
@@ -23,17 +24,27 @@ export class ReqTagRunner extends TagRunner {
   }
 
   public async run({ context, args, contentArgs }: TagRunnerOptions) {
-    const name = args[0];
+    const name = String(args[0]);
+
+    const parameterizer = new Parameterizer(
+      this.executor.prepare.bind(this.executor)
+    );
+    // parameterizer from parent, we should set it back after rendered our context.
+    const parentParameterizer = context.lookup(PARAMETERIZER_VAR_NAME);
+    context.setVariable(PARAMETERIZER_VAR_NAME, parameterizer);
```

Comment:
Because the parent and children shared the same context, if the children don't set it back, their parent will lost its parameterize because it will be overriden.